src/network/networkd-link.c

```diff
@@ -218,6 +233,31 @@ static IPv6PrivacyExtensions link_ipv6_privacy_extensions(Link *link) {
         return link->network->ipv6_privacy_extensions;
 }
 
+static int link_enable_ipv6(Link *link) {
+        const char *p = NULL;
+        bool state = false;
```

Comment:
So... state==true means disabled. Damn confusing. Please rename this to "disabled" and don't initialize, since it's set unconditionally below.